packages/web-sdk-rs/Cargo.toml

```diff
@@ -19,8 +19,8 @@ getrandom = { version = "0.2", features = ["js"] }
 hex = "0.4.3"
 iri-string = "0.7.8"
 js-sys = "0.3.59"
-tinycloud-sdk-rs = { git = "https://github.com/tinycloudlabs/tinycloud-node.git", rev = "4ac251ff" }
-tinycloud-sdk-wasm = { git = "https://github.com/tinycloudlabs/tinycloud-node.git", rev = "4ac251ff" }
+tinycloud-sdk-rs = { git = "https://github.com/tinycloudlabs/tinycloud-node.git", rev = "8d32b21" }
+tinycloud-sdk-wasm = { git = "https://github.com/tinycloudlabs/tinycloud-node.git", rev = "8d32b21" }
```

Comment:
@chunningham Update this and push to change this build to point at the latest TinyCloud Node changes

Reply:
[Hosted Demo here](https://sdk-update-0-3-0.web-sdk-693.pages.dev/)